### ISSUE TITLE
The "Bayesian Differential Equation" tutorial needs small improvements

### ISSUE THREAD

**Issue by nathanaelbosch** [TuringLang/Turing.jl#1576]
From: https://github.com/TuringLang/Turing.jl/edit/master/docs/_tutorials/10_BayesianDiffEq.md

The tutorial results seem to have changed since its initial writing. For example, the plot in the __Data retrodiction__ section does not reproduce the true ODE solution "quite accurately", since there is a big mismatch between the parameters in the three chains. I suspect that to resolve this issue and fix the example, some minor parameter tweaking could already be sufficient.

**Comment by devmotion** [comment]
Yes, the results should definitely be improved or one has to weaken this claim :smile: 

Currently, the tutorials are reorganized and moved to Julia markdown files that are easier to handle than Jupyter notebooks. You can follow the progress here: https://github.com/TuringLang/TuringTutorials/pull/113 It should be done soon and be based on more recent versions of Turing, MCMCChains etc. Hopefully the new structure makes it easier to keep the package versions updated and fix issues.
